src/main/scala/com/databricks/spark/xml/package.scala

```diff
@@ -82,69 +78,16 @@ package object xml {
     //   </fieldA>
     //
     // Namely, roundtrip in writing and reading can end up in different schema structure.
+    @deprecated("Use DataFrameWriter.write()", "0.4.0")
     def saveAsXmlFile(
         path: String, parameters: Map[String, String] = Map(),
         compressionCodec: Class[_ <: CompressionCodec] = null): Unit = {
-      val options = XmlOptions(parameters.toMap)
```

Comment:
I moved this to src/main/scala/com/databricks/spark/xml/util/XmlFile.scala below.